### PR TITLE
180 bricks connected to core influence position measurement of core

### PR DESCRIPTION
**Pull request by surgura**

### CODE REVIEW

File: runners/mujoco/setup.py [hunk 1]
```diff
@@ -20,7 +20,7 @@
     install_requires=[
         f"revolve2-core @ file://{os.path.join(revolve2_path, 'core')}",
         "mujoco>=2.2.0",
-        "mujoco-python-viewer @ git+https://github.com/rohanpsingh/mujoco-python-viewer@d61433f3991294da455751659925839452b9597e",
+        "mujoco-python-viewer>=0.1.2",
         "dm-control>=1.0.3",
         "opencv-python>=4.6.0.66",
```
Comment (tckarenchiang) [inline runners/mujoco/setup.py:25]:
```suggestion
        "opencv-python>=4.6.0.66",
        "pyyaml>=6.0",
```

Comment (tckarenchiang) [inline runners/mujoco/setup.py:25]:
```suggestion
        "opencv-python>=4.6.0.66",
        "pyyaml>=6.0",
```

File: core/revolve2/core/modular_robot/_body.py [hunk 2]
```diff
@@ -156,6 +158,17 @@ def to_grid(
         """
         return _GridMaker().make_grid(self)
 
+    def body_state_from_actor_state(self, actor_state: ActorState) -> BodyState:
```
[the remark this answers is not in the document]
Comment (tckarenchiang) [reply core/revolve2/core/modular_robot/_body.py:161]: Oh I just see it now: https://github.com/ci-group/revolve2/issues/231

File: runners/mujoco/setup.py [hunk 3]
```diff
@@ -20,7 +20,7 @@
     install_requires=[
         f"revolve2-core @ file://{os.path.join(revolve2_path, 'core')}",
         "mujoco>=2.2.0",
-        "mujoco-python-viewer @ git+https://github.com/rohanpsingh/mujoco-python-viewer@d61433f3991294da455751659925839452b9597e",
+        "mujoco-python-viewer>=0.1.2",
```
[the remark this answers is not in the document]
Comment (surgura) [reply runners/mujoco/setup.py:23]: I do not have this issue. when I install only runners/mujoco and standard_resources, `yaml` is not installed and simulate mujoco example works without issues

**Review by tckarenchiang — APPROVED** [review]

**Pull request merged**